.github/workflows/emulate.yml

```diff
@@ -6,14 +6,20 @@ on:
 
 jobs:
   check:
-    runs-on: ubuntu-latest
+    runs-on: macos-latest
```

Comment:
Is there a reason for the change from Ubuntu to macOS?

This is ground we've covered before, where we intentionally changed from macOS to Ubuntu for a couple of reasons:

1. Test run stability
2. GitHub resource constraints

Reply:
Yes there is, not all emulators are available in Ubuntu as macOS has HAXM installed so we are able to create a new AVD instance, launch an emulator with hardware acceleration, and run our Android tests directly on the VM.
This is described [here](https://github.com/ReactiveCircus/android-emulator-runner#github-action---android-emulator-runner).

Reply:
sure, we will keep an eye on it.